httpx/concurrency/asyncio.py

```diff
@@ -194,6 +194,44 @@ def loop(self) -> asyncio.AbstractEventLoop:
             stream_reader=stream_reader, stream_writer=stream_writer, timeout=timeout
         )
 
+    async def start_tls(
+        self,
+        stream: BaseStream,
+        hostname: str,
+        ssl_context: ssl.SSLContext,
+        timeout: TimeoutConfig,
+    ) -> BaseStream:
+
+        loop = self.loop
+        if not hasattr(loop, "start_tls"):  # pragma: no cover
+            raise NotImplementedError(
+                "asyncio.AbstractEventLoop.start_tls() " "available in Python 3.7+"
```

Comment:
Just a bit of phrasing here:

```suggestion
                "asyncio.AbstractEventLoop.start_tls() is only available in Python 3.7+"
```